.github/workflows/vineyard-operator.yaml

```diff
@@ -122,7 +122,13 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-
+      
+      - name: Build a kind cluster with local registry
+        run: |
+          touch /tmp/e2e-k8s.config
+          export KUBECONFIG=/tmp/e2e-k8s.config
+          bash k8s/test/hack/build-kind-cluster-with-local-registry.sh
```

Comment:
I suggest moving the "install cert-manager" and "deploying vineyardd cluster" steps to here as well to further reduce the duplications in "xxxx/e2e.yaml"s.

Reply:
Make sense